Architect-Coders/MU8
# Issue #12: Create Characters Activity

**What's in:**

- Organized modules a little better.
- Created Characters MVVM, Use Case and Repository.
- Added Coroutines to get the characters from the web service.

**Testing Done:**

* On real device: Pixel 2 XL.

**How it looks:**

https://drive.google.com/open?id=1htO0pjIt1jycLYPyn8cn8bmmodP4lQ6N

**How to test:**

* Launch the app and reach the Categories screen.
* Tap on Characters option and see the new Activity List.

**Signed-off-by:**

* Diego Recalde <difereto89@gmail.com>
